client/app/bundles/course/group/forms/NameDescriptionForm.jsx

```diff
@@ -32,61 +32,79 @@ const styles = {
     width: '100%',
   },
 };
+const validationSchema = yup.object({
+  name: yup.string().required(formTranslations.required),
```

Comment:
I think you dropped the name length validation (< 255).

Reply:
Re-added!